CharacterCreation/06_01_Systemized_Skills.txt

```diff
@@ -0,0 +1,117 @@
+==============================
+Full list of skills

```

Comment:
I believe this should stay in DEV until complete -- and until it replaces the existing skills list. C2.0 will have issues with this current writing. Also there are lots of dev notes here, some of which digress from the direction of discussions I remember back in December. We should discuss this material -- lots of interesting things in here.